cocos/renderer/gfx/IndexBuffer.cpp

```diff
@@ -110,11 +111,25 @@ void IndexBuffer::update(uint32_t offset, const void* data, size_t dataByteLengt
     {
 //        glBufferData(GL_ELEMENT_ARRAY_BUFFER, (GLsizeiptr)dataByteLength, data, glUsage);
         glBufferData(GL_ELEMENT_ARRAY_BUFFER, _bytes, (const GLvoid*)data, glUsage);
+        GLenum gl_err = glGetError();
+        if (0 != gl_err){
+            RENDERER_LOGE("GL error 0x%x: %s:%s", gl_err, glEnumName(gl_err), __FUNCTION__);
+            _drawable = false;
+        }else{
+            _drawable = true;
+        }
         _needExpandDataStore = false;
     }
     else
     {
         glBufferSubData(GL_ELEMENT_ARRAY_BUFFER, (GLintptr)offset, (GLsizeiptr)dataByteLength, (const GLvoid*)data);
+        GLenum gl_err = glGetError();
+        if (0 != gl_err){
+            RENDERER_LOGE("GL error 0x%x: %s:%s", gl_err, glEnumName(gl_err), __FUNCTION__);
+            _drawable = false;
+        }else{
+            _drawable = true;
+        }
```

Comment:
Duplicate codes. I know may want to log the function line, but can use `_needExpandDataStore ` to know which codes are executed.

Reply:
Actually both blocks will set `_needExpandDataStore` as false after execution. 
But it's fine to catch them all in one piece of code. I will modify it a little bit.